mwdb/web/src/components/Settings/Views/GroupMembers.js

```diff
@@ -1,20 +1,155 @@
+    const [modalSpec, setModalSpec] = useState({});
+
+    function setAdmin(member, membership) {
+        let message = `Are you sure to change admin group permissions for ${member}?`;
+
+        setModalSpec({
+            message,
+            action: () => {
+                setModalOpen(false);
+                setAdminMembership(member, membership);
+            },
+            buttonStyle: "bg-success",
+            confirmText: "Yes",
+        });
+        setModalOpen(true);
+    }
+
+    function removeMembership(member) {
+        let message = `Remove ${member} from this group`;
+
+        setModalSpec({
+            message,
+            action: () => {
+                setModalOpen(false);
+                removeMember(member);
+            },
+            confirmText: "Remove",
+        });
+        setModalOpen(true);
+    }
+
+    return (
+        <React.Fragment>
+            {members.map((member) => (
+                <tr>
+                    <th className="col">
+                        <span className="badge badge-info">{member.login}</span>
+                        {admins.includes(member.login) ? (
+                            <span
+                                data-toggle="tooltip"
+                                title="Group admin user"
+                            >
+                                <FontAwesomeIcon
+                                    className="navbar-icon"
+                                    icon={faCrown}
+                                />
+                            </span>
+                        ) : (
+                            []
+                        )}
+                    </th>
+                    <td className="col-auto">
+                        <Link
+                            data-toggle="tooltip"
+                            title="Remove user from group"
+                            onClick={(ev) => {
+                                ev.preventDefault();
+                                removeMembership(
+                                    member.login,
+                                    !admins.includes(member.login)
+                                );
+                            }}
+                        >
+                            <FontAwesomeIcon icon={faTrash} />
+                        </Link>
+                    </td>
+                    <td className="col-auto">
+                        <Link
+                            data-toggle="tooltip"
+                            title="Give or revoke group admin permissions"
+                            onClick={(ev) => {
+                                ev.preventDefault();
+                                setAdmin(
+                                    member.login,
+                                    !admins.includes(member.login)
+                                );
+                            }}
+                        >
+                            <FontAwesomeIcon icon={faUser} />
```

Comment:
That button is more intuitive with crown icon. For me, user icon suggests user details.
```suggestion
                            <FontAwesomeIcon icon={faCrown} />
```